packages/components/src/components/assets/CoinLogo/coins.ts

```diff
@@ -20,4 +20,5 @@ export const COINS = {
     vtc: require('../../../images/coins/vtc.svg'),
     xrp: require('../../../images/coins/xrp.svg'),
     zec: require('../../../images/coins/zec.svg'),
+    sol: require('../../../images/coins/ada.svg'), // TODO(vl): add icons
```

Comment:
I am aware this is not the right way to add a new icon, as you can see we use ada icon for solana for now, expect this to be resolved in separate PR.